### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by Ivans02** [UltimaHoarder/UltimaScraper#1859] [Q&A]
Hello!

I'm trying to run the code, but in PowerShell appear this:
Site: 0 = onlyfans |

It's working? I do something wrong?

Thanks
![erro](https://user-images.githubusercontent.com/91340613/161438955-3bb41c26-59d0-4231-a325-a81df7e5ca95.png)

**Comment by Lichterflug** [comment]
What is it doing after you entered "0" and pressed ENTER?
